resources/var/tmp/scm/init.script.d/060-cas-configuration.groovy

```diff
@@ -3,22 +3,23 @@
 import groovy.json.JsonSlurper;
 
 // TODO sharing ?
-def getValueFromEtcd(String key){
-  String ip = new File("/etc/ces/node_master").getText("UTF-8").trim();
-	URL url = new URL("http://${ip}:4001/v2/keys/${key}");
-	def json = new JsonSlurper().parseText(url.text)
-	return json.node.value
+def getValueFromEtcd(String key) {
+    String ip = new File("/etc/ces/node_master").getText("UTF-8").trim();
+    URL url = new URL("http://${ip}:4001/v2/keys/${key}");
+    def json = new JsonSlurper().parseText(url.text)
+    return json.node.value
 }
 
 try {
-    def cas = injector.getInstance( Class.forName("com.cloudogu.scm.cas.CasContext", true, Thread.currentThread().getContextClassLoader()) );
-	def config = cas.get();
+    def cas = injector.getInstance(Class.forName("com.cloudogu.scm.cas.CasContext", true, Thread.currentThread().getContextClassLoader()));
+    def config = cas.get();
 
-	String fqdn = getValueFromEtcd("config/_global/fqdn");
-	config.setCasUrl("https://${fqdn}/cas");
-	config.setEnabled(true);
+    String fqdn = getValueFromEtcd("config/_global/fqdn")
+    config.setCasUrl("https://${fqdn}/cas")
+    config.setEnabled(true)
+    config.setAllowedProxyChains("^https://${fqdn}/.*\$")
```

Comment:
I think we should escape regex meta characters here e.g.: `192.168.56.2` should be converted to `192\.168\.56\.2`. Because `192.168.56.2` would match `192-168-56-2` too.